proto/xudp2p.proto

```diff
@@ -0,0 +1,116 @@
+syntax = "proto3";
+
+import "annotations.proto";
```

Comment:
Is this line actually needed? I thought it was just for the http metadata, which it doesn't seem like we are using here.

Reply:
Yes, forgot to remove this one.